### PR TITLE
SVG sprite system implementation

### PR DESCRIPTION
**Pull request by junan**
## What happened

Fixed [23](https://github.com/junan/nimble-survey/issues/23)

## Insight

Used [svg-icon-sprite](https://github.com/jannicz/svg-icon-sprite) for svg sprite

## Proof of Work

Bu adding the following html in the `app.component.html`
```html
<img src="assets/images/logo.svg" />
<img src="assets/images/success.svg" />
<img src="assets/images/smile-sunglass.svg" />
<img src="assets/images/user-picture.svg" />
<img src="assets/images/thumb.svg" class="icon" />
<img src="assets/images/action-right.svg" class="icon icon--lg" />
<svg-icon src="assets/images/icon-sprite.svg#arrow-left" class="icon"></svg-icon>
<svg-icon src="assets/images/icon-sprite.svg#arrow-right" class="icon"></svg-icon>
<svg-icon src="assets/images/icon-sprite.svg#action-right" class="icon"></svg-icon>
<svg-icon src="assets/images/icon-sprite.svg#close" class="icon"></svg-icon>
<svg-icon src="assets/images/icon-sprite.svg#error" class="icon"></svg-icon>
<svg-icon src="assets/images/icon-sprite.svg#notification" class="icon"></svg-icon>

it is showing the icons

```
Screen Shot 2020-11-27 at 2 20 25 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,3 +25,12 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
 ## Deployment
 
 Deploy to Heroku for [staging](https://nimble-survey-staging.herokuapp.com/) and [production](https://nimble-survey-production.herokuapp.com/) release
+
+### SVG icons
+
+This application uses `svg-icon-sprite` npm package to generate icon sprite.Icon sprite located at `assets/images/icon-sprite.svg` which generated from SVG icons in `assets/images/icons`.
```
Comment (Lahphim) [inline README.md:31]: @junan is this typo `sprite.Icon` ? 🤔

Comment (junan) [reply README.md]: Added extra space

**Review by Lahphim — APPROVED** [review]

File: src/assets/images/action-right.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<svg width="56" height="56" viewBox="0 0 56 56" fill="none" xmlns="http://www.w3.org/2000/svg">
+<circle cx="28" cy="28" r="28" fill="white"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M24.4629 38.0827L33.7078 28.9458C34.0974 28.5612 34.0974 27.9398 33.7078 27.5542L24.4629 18.4173C23.9005 17.8609 22.9856 17.8609 22.4223 18.4173C21.86 18.9738 21.86 19.8769 22.4223 20.4333L30.3309 28.2505L22.4223 36.0657C21.86 36.6231 21.86 37.5262 22.4223 38.0827C22.9856 38.6391 23.9005 38.6391 24.4629 38.0827Z" fill="#15151A"/>
```
Comment (carryall) [inline src/assets/images/action-right.svg:3]: @junan I'm not sure about this but do we have to change any `fill` color in this icon to `currentColor` too so we can set the color by styling later?

**Review by carryall — APPROVED** [review]

File: src/assets/images/action-right.svg (hunk 2 above)
Comment (junan) [reply src/assets/images/action-right.svg]: Updated to `currentColor` for now

**Pull request merged**